pkg/cloudprovider/types.go

```diff
@@ -268,3 +268,26 @@ func IgnoreNodeClassNotReadyError(err error) error {
 	}
 	return err
 }
+
+// RetryableError is an error type returned by CloudProviders when the action emitting the error has to be retried
+type RetryableError struct {
```

Comment:
Not sure that we have a need for using a `RetryableError` if we can just re-leverage our NodeClaimNotFound error